snippets/logging-http-sys-input-support.adoc

```diff
@@ -0,0 +1,19 @@
+// Text snippet included in the following assemblies:
+//
+// Text snippet included in the following modules:
+// * log-collector-rsyslog-server
+// * 
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+{feature-name} is only supported for the following scenarios:
+
+* Logging is installed on {hcp-capital}. 
+* When logs originate from a Red{nbsp}Hat-supported product that is installed on the same cluster as the {CLO}. For example:
+ifeval::["{feature-name}" == "Syslog receiver input"]
+** Red Hat OpenStack on OpenShift
```

Comment:
Should this be an attribute:
`:rhoso-first: Red Hat OpenStack Services on OpenShift (RHOSO)`

Reply:
i can't find :rhoso-first: in common-attributes.adoc